packages/web3-providers-ws/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   `main` and `files` entries in `package.json` changed to `lib/` directory from `dist/` (#5739)
 -   Refactor to use common SocketProvider class (#5683)
+-   Legacy Event `close` has been deprecated, superseded by `disconnect` (#5683)
```

Comment:
This and other changelog should be under unreleased

Reply:
This change happened in 4.0.1-alpha.4 but it was just undocumented, should it still be under unreleased?

Reply:
I would think it makes sense to leave it under `4.0.1-alpha.4`, but we should also update the release notes for the [Github release](https://github.com/web3/web3.js/releases/tag/v4.0.1-alpha.4)